File: SAP_APIv1/SAP_API/DTOs/Responses/User/PreparedBakingProgramResponse.cs


using SAP_API.Models;
using System;

namespace SAP_API.DTOs.Responses.User
{
    public class PreparedBakingProgramResponse
    {
        public Guid Id { get; set; }
        public string Code { get; set; }
        public DateTime CreatedAt { get; set; }
        public BakingProgramStatus Status { get; set; }
        public int BakingTimeInMins { get; set; }
        public int BakingTempInC { get; set; }
        public DateTime BakingProgrammedAt { get; set; }
        public DateTime? BakingStartedAt { get; set; }
        public Guid OvenId { get; set; }
        public string OvenCode { get; set; }
    }
}

File: bakery-app/src/models/Responses/User/PreparedBakingProgramResponse.ts
import { BakingProgramStatus } from "../../Enums/BakingProgramStatus";

export default interface PreparedBakingProgramResponse {
    id: string;
    code: string;
    createdAt: string;
    bakingProgramStatus: BakingProgramStatus;
    bakingTimeInMins: number;
    bakingTempInC: number;
    bakingProgrammedAt: string;
    bakingStartedAt: string;
    ovenId: string;
    ovenCode: string;
};